<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="maskOne">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Maske</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintOne">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inpaint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageOne">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eksempel 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="glattingOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Inpainting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>**Tekst om funksjonalitet**</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintingCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QLabel" name="inpaintImg">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../hdr-bilder/Tree/Tree_00032.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="maskTwo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Maske</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageTwo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eksempel 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintTwo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inpaint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maskThree">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Maske</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageThree">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eksempel 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inpaintThree">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Inpaint</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
